spec/controllers/account/projects_controller_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe Account::ProjectsController, type: :controller do
+  render_views
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:project) { create(:project, workspace: workspace) }
+  let!(:project_valid_params) { { name: "test_project"} }
+  let!(:project_invalid_params) { { name: nil} }
+
+  before do
+    sign_in user
+  end
+
+  describe "GET #index in Account/Projects controller" do

```

Comment:
we do not need `Account/Projects controller` in description those specs are already for it